cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/KafkaCruiseControl.java

```diff
@@ -496,31 +496,13 @@ public void setRequestedExecutionProgressCheckIntervalMs(Long requestedExecution
   }
 
   /**
-   * Dynamically set the inter-broker partition movement concurrency per broker.
+   * Dynamically set the execution concurrency per broker.
    *
-   * @param requestedInterBrokerPartitionMovementConcurrency The maximum number of concurrent inter-broker partition movements
-   *                                                         per broker.
+   * @param requestedExecutionConcurrency The maximum number of concurrent execution per broker.
+   * @param concurrencyType The type of the execution concurrency to change
    */
-  public void setRequestedInterBrokerPartitionMovementConcurrency(Integer requestedInterBrokerPartitionMovementConcurrency) {
```

Comment:
Given the lack of class-level javadocs, I can't tell: are these added and removed APIs public in the sense of being usable by external software, or are they used purely within CC itself?

Reply:
It's only used within CC itself.